KomaMRIPlots/src/ui/DisplayFunctions.jl

```diff
@@ -154,38 +154,38 @@ julia> plot_seq(seq)
 ```
 """
 function plot_seq(
-      seq::Sequence; 
-      width=nothing, 
-      height=nothing, 
+      seq::Sequence;
+      width=nothing,
+      height=nothing,
       slider=false,
-      show_seq_blocks=false, 
-      darkmode=false, 
-      max_rf_samples=Inf, 
-      range=[], 
-      title="", 
+      show_seq_blocks=false,
+      darkmode=false,
+      max_rf_samples=Inf,
+      range=[],
+      title="",
       xaxis="x",
-      yaxis="y", 
+      yaxis="y",
       showlegend=true
   )
 	idx = ["Gx" "Gy" "Gz"]
 	N = length(seq)
 	O = size(seq.RF,1)
-	ΔT = KomaMRICore.durs(seq)
+	ΔT = seq.DUR
 	T0 = cumsum([0; ΔT],dims=1)
```

Comment:
We do this so many times, that we could add a function `get_block_start`  or something like that.

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/241/commits/5998d0fe80082ea26f62b0efa53cfe4a9fe8c1a0